docs/FAQ/FAQ-General.md

```diff
@@ -101,7 +101,18 @@ If you have an issue paying a BTCPay Server invoice, it is likely do to one of t
 
 1. You are trying to pay using a non-segwit wallet and the merchant's invoices use Bech32 format.
 
-This is a fairly common issue but can be confusing to the user who will likely receive wallet errors similar to `invalid address` when making a payment to an invoice. The solution for this (by the customer) is to use a [SegWit compatible wallet](https://en.bitcoin.it/wiki/Bech32_adoption) which supports sending to Bech32 addresses. The solution for this (by the merchant) is to [modify your extended public key (xpub)](./FAQ-Wallet.md#what-is-a-derivation-scheme) you are providing in your BTCPay Server store to use the format `-[p2sh]` which will wrap your addresses with a Pay to Script Hash and allow both SegWit and non-SegWit wallets to make payments to your addresses. 
+This is a fairly common issue but can be confusing to the user who will likely receive wallet errors similar to `invalid address` when making a payment to an invoice. The solution for this (by the customer) is to use a [SegWit compatible wallet](https://en.bitcoin.it/wiki/Bech32_adoption) which supports sending to Bech32 addresses. 
+
+The solution for this (by the merchant) is to modify your extended public key (xpub) you are providing in your BTCPay Server store. To do this, you can append your xpub with `-[p2sh]` which will automatically modify your invoice addresses to allow both SegWit and non-SegWit wallets to make payments to your addresses. BTCPay Server wallet will do this by wrapping the xpub's addresses with a Pay to Script Hash (p2sh) which produces more widely accepted addresses. It's important to understand how this can affect your wallet and the payments received before and after implementing this solution in your BTCPay Server store. Modifying your store's xpub will generate an entirely new wallet from the perspective of your BTCPay Server store. Please understand the following before blindly implementing the solution:
```

Comment:
```suggestion
The solution for this (by the merchant) is to modify your extended public key (xPub) you are providing in your BTCPay Server store. To do this, you can append your xPub with `-[p2sh]` which will automatically modify your invoice addresses to allow both SegWit and non-SegWit wallets to make payments to your addresses. The BTCPay Server wallet will do this by wrapping the xPub's addresses with a Pay to Script Hash (p2sh) which produces more widely accepted addresses. It's important to understand how this can affect your wallet and the payments received before and after implementing this solution in your BTCPay Server store. Modifying your store's xPub will generate an entirely new wallet from the perspective of your BTCPay Server store. Please understand the following before blindly implementing the solution:
```